src/App.tsx

```diff
@@ -1,11 +1,16 @@
 import React from "react";
 import "./App.css";
```

Comment:
Remove app.css

Reply:
it is used for theme i.e to center the number

Reply:
use material-ui libraray
makeStyles

Reply:
will it be used after theme provider implementation?

Reply:
you can use any time for alignment and all...